tests/core/data/test_csv_data_node.py

```diff
@@ -179,6 +196,29 @@ def test_write(self, csv_file, default_data_frame, content, columns):
         csv_dn.write(None)
         assert len(csv_dn.read()) == 0
 
+    @pytest.mark.parametrize(
+        "content,columns",
+        [
+            ([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}], None),
+            ([[11, 22, 33], [44, 55, 66]], None),
+            ([[11, 22, 33], [44, 55, 66]], ["e", "f", "g"]),
+        ],
+    )
+    def test_write_modin(self, csv_file, default_data_frame, content, columns):
+        default_data_frame = modin_pd.DataFrame(default_data_frame)
+        csv_dn = CSVDataNode("foo", Scope.PIPELINE, properties={"path": csv_file, "exposed_type": "modin"})
+        assert np.array_equal(csv_dn.read().values, default_data_frame.values)
+        if not columns:
+            csv_dn.write(content)
+            df = pd.DataFrame(content)
```

Comment:
For consistency matter, it should use modin_pd:
```suggestion
            df = modin_pd.DataFrame(content)
```